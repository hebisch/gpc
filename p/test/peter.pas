program t( output , input );
    begin
        while not eof do begin
            while not eoln do begin
                if input^ in [ '+' , '-' , '*' , '/' , '%' ] then
                    case input ^ of
                        '+':    writeln( 'plus' );
                        '-':    writeln( 'minus' );
                        '*':    writeln( 'star' );
                        '/':    writeln( 'slash' );
                        '%':    writeln( 'percent' );
                    end
                else
                    writeln( 'random char' );
                get(input);
            end;
            get(input);
        end;
    end.
