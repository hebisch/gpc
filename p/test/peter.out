plus
random char
minus
random char
star
random char
slash
random char
percent
